doc/cascadia/profiles.schema.json

```
@@ -54,6 +54,23 @@
         }
       ]
     },
+    "BellSound": {
+      "default": "",
+      "description": "Sets the file location of the sound played when the application emits a BEL character. If the path is invalid no sound will be played. This property also accepts an array of sounds and the terminal will pick one at random.",
+      "pattern": "",
+      "oneOf": [
+      {
+        "type": "string",
+        "type": "null"
```

```suggestion
        "type": [
          "string",
          "null"
        ]
```
`backgroundImage` has it like this, so I'm gonna suggest that it look like this haha.